### DISCUSSION TITLE
SDK 1.8.0 | Translations missing - Spanish

### DISCUSSION THREAD

**Discussion by rnm-vortal** [OP-TED/eForms-SDK#641] [Q&A] [answered]
Hi,

We noticed that in v1.8.0 the translations for the following groups are missing in Spanish:
GR-PREVIOUS-NOTICE (ND-PreviousNoticeReference)
GR-PROCEDURE-ADDITIONAL-NATURE (ND-ProcedureContractAdditionalNature)
GR-GROUP-COMPOSITION-REFERENCE (ND-GroupCompositionLotReference)
GR-LOT-ADDITIONAL-NATURE (ND-LotContractAdditionalNature)
GR-LOT-ENVIRONMENTAL-IMPACT (ND-LotEnvironmentalImpactType)
GR-LOT-SOCIAL-OBJECTIVE (ND-LotSocialObjectiveType)
GR-LOT-INNOVATION-PROCUREMENT (ND-LotInnovativeAcquisitionType)
GR-LOT-SELECTIONCRITERIA-SECONDSTAGEINVITE (ND-SecondStageCriterionParameter)
GR-LOT-AWARDCRITERIA-WEIGHTCRITERION-PARAMETER (ND-LotAwardWeightCriterionParameter)
GR-LOT-AWARDCRITERIA-FIXEDCRITERION-PARAMETER (ND-LotAwardFixedCriterionParameter)
GR-LOT-AWARDCRITERIA-THRESHOLDCRITERION-PARAMETER (ND-LotAwardThresholdCriterionParameter)
GR-LOT-SUBMISSION LANGUAGE (ND-LotSubmissionLanguage)
GR-ORGANISATION-IDENTIFIER (ND-CompanyLegalEntity)

Will this be corrected?
Thank you.

**Comment by rouschr** [comment, marked as answer]
Hello,
in the data for SDK 1.9 I see a label with ID "group|name|ND-PreviousNoticeReference" and it has an "es" translation "Referencia de aviso anterior". So it looks like SDK 1.9 will have these translations.
Note that the notice-types provides the label ID for each item via "_label"
